packages/zapp/console/src/components/Launch/LaunchForm/constants.ts

```diff
@@ -45,6 +45,7 @@ export const typeLabels: { [k in InputType]: string } = {
   [InputType.Schema]: 'schema - uri',
   [InputType.String]: 'string',
   [InputType.Struct]: 'struct',
+  [InputType.Union]: '',
```

Comment:
do we need some string here as 'union type' . If not - maybe you can add comment to why not need.

Reply:
just added it and update the formatType function https://github.com/flyteorg/flyteconsole/blob/563950a8f0a465a5495c7b5769bbffd218e43379/packages/zapp/console/src/components/Launch/LaunchForm/utils.ts#L78-L93